LeetCode/medium/remove_nth_from_end_19.py

```
@@ -0,0 +1,30 @@
+from typing import Optional
+
+
+class ListNode:
+    def __init__(self, val=0, next=None):
```

Ruff (0.15.18)

[error] Function argument `next` is shadowing a Python builtin

(A002)